src/k2/dto/MessageVerbs.h

```diff
@@ -38,6 +38,11 @@ enum Verbs : k2::Verb {
     CPO_PERSISTENCE_CLUSTER_CREATE,
     // ControlPlaneOracle: asked to return a collection of persistence servers
     CPO_PERSISTENCE_CLUSTER_GET,
+    PERSISTENCE_CLUSTER_CREATE,
+    PERSISTENCE_CLUSTER_GET,
+    CPO_METADATA_PERSIST,
```

Comment:
CPO_LOGSTREAM_METADATA_PUT
and
CPO_LOGSTREAM_METADATA_GET

Reply:
changed